dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/internal/handlers/AndroidLogsHandler.kt

```diff
@@ -0,0 +1,40 @@
+package com.datadog.android.tracing.internal.handlers
+
+import com.datadog.android.log.Logger
+import datadog.opentracing.LogsHandler
+
+internal class AndroidLogsHandler(
+    val logger: Logger
+) : LogsHandler {
+
+    // region Span
+    override fun log(timestampMicroseconds: Long, fields: Map<String, *>) {
+        // not sure that we should do something with the timestamp here ???
```

Comment:
In theory we should be able to override the timestamp by adding a `"date"` attribute with the properly formatted timestamp (cf `LogSerializer`). 
The idea is that the log should have the given timestamp, and not the current one.